internal/utils/helper.go

```diff
@@ -107,28 +109,36 @@ func CompareGitUrl(remoteGitUrl string) bool {
 //Special ports are specified for catalogs since the ports for them are not in buildconfig.
 //if the service has either of the given names occurring in the service name, it will return the specified port
 //otherwise will fetch the port from buildconfig and return it.
-func GetBlockPort(block *types.Block) int {
+func GetBlockPort(blockName string, release *types.Release) int {
 
-	if strings.Contains(block.Name, "redis") {
+	if strings.Contains(blockName, "redis") {
 		return config.RedisPort
-	} else if strings.Contains(block.Name, "postgres") {
+	} else if strings.Contains(blockName, "postgres") {
 		return config.PostgresPort
-	} else if strings.Contains(block.Name, "mongodb") {
+	} else if strings.Contains(blockName, "mongodb") {
 		return config.MongoPort
-	} else if strings.Contains(block.Name, "minio") {
+	} else if strings.Contains(blockName, "minio") {
 		return config.MinioPort
-	} else if strings.Contains(block.Name, "mysql") {
+	} else if strings.Contains(blockName, "mysql") {
 		return config.MysqlPort
 	} else {
-		resp := GetLatestSuccessfulRelease(block.Releases).RunConfig.Port
-		port, err := strconv.Atoi(resp)
+		port, err := strconv.Atoi(release.RunConfig.Port)
 		if err != nil {
 			TerminateWithError(err)
 		}
 		return port
 	}
 }
 
+func CanPortForwardToRelease(release *types.Release) bool {
+	if release.RunConfig != nil &&
+		(release.RunConfig.Type == types.Block_BACKEND_API || release.RunConfig.Type == types.Block_CATALOG) {
```

Comment:
shouldn't we be allowing forwarding for web-apps also?

Reply:
I am not sure, why would anyone make a website private? we could consider it if someone ask for it but I don't see the use case